packages/frontend/src/page/hooks/useWagmiPassiveHook.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+
+interface HookParams {
+    enabled?: boolean;
+    [key: string]: unknown;
+}
+
+interface UsePassiveHookParams<T> {
+    hook: (params: HookParams) => T;
+    params: HookParams;
+}
+
+export function useWagmiPassiveHook<T>({
```

Comment:
Very interesting. I think this would actually be great to have in the React library. And it could be interesting to actually export all the `wagmi` functions from there so that they automatically follow this logic. But for now just moving this to the React lib would be good.

Reply:
Oh ye, it makes sense, I'll do that.